openhcl/hcl/src/protocol.rs

```diff
@@ -250,7 +264,10 @@ pub struct tdx_vp_context {
     pub pad2: [u8; 32],
     pub entry_rcx: x86defs::tdx::TdxVmFlags,
     pub gpr_list: x86defs::tdx::TdxL2EnterGuestState,
-    pub pad3: [u8; 96],
+    pub ipi_offload_irr: [u32; 8],
```

Comment:
Actually, this is kind of dodgy. Elsewhere, you get a `&mut` to this whole context, but the kernel is still aliasing `ipi_offload_irr` from other processors.

For this to be correct, you probably have to stop handing out `&mut` to the whole context and instead use `addr_of`/`offset_of` to get just the pieces that are safe to touch.